curriculum/challenges/english/03-front-end-libraries/jquery/change-text-inside-an-element-using-jquery.english.md

```diff
@@ -12,7 +12,7 @@ Here's how you would rewrite and emphasize the text of our heading:
 <code>$("h3").html("&#60;em&#62;jQuery Playground&#60;/em&#62;");</code>
 jQuery also has a similar function called <code>.text()</code> that only alters text without adding tags. In other words, this function will not evaluate any HTML tags passed to it, but will instead treat it as the text you want to replace the existing content with.
 Change the button with id <code>target4</code> by emphasizing its text.
-Check this <a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">link</a> to know more on the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62;</code> and their uses.
+Check this link to <a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">go to the MDN web docs for <code>&#60;em&#62;</code></a> to learn the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62</code> and their uses.
```

Comment:
@RobertoJBeltran You actually did not make the change @co16353sidak suggested.  Also, after reviewing this myself, I suggest the following change:
```suggestion
<a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">View the MDN web docs for &#60;em&#62;</a> to learn the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62</code> and their uses.
```

Reply:
@RobertoJBeltran I agree with @RandellDawson here, this suggestion seems much better to me